app/test_main.py

```diff
@@ -1,3 +1,47 @@
-from app.main import get_human_age
+import pytest
 
-# write your code here
+from .main import get_human_age
+
+
+def test_add_not_int_argument() -> None:
+    with pytest.raises(TypeError):
+        get_human_age(12.52, "52.12")
```

Comment:
The test `test_add_not_int_argument` correctly checks for a `TypeError` when non-integer arguments are passed. Ensure that the `get_human_age` function indeed raises a `TypeError` for such cases.